modules/large-deployments/pages/tuning.adoc

```diff
@@ -432,7 +432,7 @@ This section contains information about the available parameters.
 |===
 | Description          | The amount of memory allocated by PostgreSQL every time a connection needs to do a sort or hash operation.
                          Every connection (as specified by xref:max-connections[`max_connections`]) might make use of an amount of memory equal to a multiple of `work_mem`.
-| Tune when            | Individual query operations are too slow, and value is below 5 MB
+| Tune when            | Database operations are slow because of excessive temporary file disk I/O. To test if that phenomenon is happening, add `log_temp_files = 5120` to [path]``/var/lib/pgsql/data/postgresql.conf``, restart Postgres, and monitor Postgres log files. If lines appear containing the string `LOG:  temporary file:` then raising this parameter's value might help reducing disk I/O thus speeding up database operations.
```

Comment:
```suggestion
| Tune when            | Database operations are slow because of excessive temporary file disk I/O. 
To test if that is happening, add ``log_temp_files = 5120`` to [path]``/var/lib/pgsql/data/postgresql.conf``, restart Postgres, and monitor the Postgres log files.
If you see lines containing ``LOG:  temporary file:`` try raising this parameter's value to help reduce disk I/O and speed up database operations.
```